### ISSUE TITLE
Console compilation error in newest commit

### ISSUE THREAD

**Issue by meijeru** [red/red#2376]
I downloaded `red-21dec16-ae52ffe.exe` and compiled it in `C:\Program Files (x86)`. Here is the result:
````
Compiling Red GUI console...
*** Loading Error: file access error: %draw-d2d.reds
*** in file: %/C/Program%20Files%20%28x86%29/Red/modules/view/backends/windows/button.reds
*** at line: 46
```

**Comment by meijeru** [comment]
Interestingly, the line with the error is NOT in `%button.reds` but in `%gui.reds`

**Comment by meijeru** [comment]
I notice it has been fixed, but the error message still needs attention ;-)

**Comment by dockimbel** [comment]
The error message is correct, the file was not included in the Red binary, so the compiler could not access it.

**Comment by meijeru** [comment]
Should thisnotbe closed? It would improve your record for 0.6.2 ("only" 21 bugs left 😄 )

**Comment by dockimbel** [comment]
@meijeru As author of a ticket, you should be the one closing it when it has been processed. One less thing for us to do. ;-)
